tools/pipelines/templates/include-test-real-service.yml

```diff
@@ -396,6 +431,30 @@ stages:
             condition: always()
             continueOnError: true # Keep running subsequent tasks even if this one fails (e.g. the tinylicious log wasn't there)
 
+        # Only release tenants that were checked out from the TRIPS tenant pool for ODSP tests
+        - ${{ if eq(parameters.stageId, 'e2e_odsp') }}:
+          # Release the tenant back to the TRIPS tenant pool
+          - task: AzureCLI@2
+            displayName: 'Log in to release tenant credentials'
+            inputs:
+              azureSubscription: 'Fluid Framework Tests'
+              addSpnToEnvironment: true
+              scriptType: 'bash'
+              scriptLocation: 'inlineScript'
+              inlineScript: |
+                set -eu -o pipefail
+
+                az login --service-principal -u $servicePrincipalId -p $idToken --tenant $tenantId
```

Comment:
isn't this the same job? why do we need to log in again? is the token expiry something like 1h and our tests run beyond that?